src/Authentication/Authenticators/AccessTokens.php

```diff
@@ -115,6 +115,8 @@ public function check(array $credentials): Result
             ]);
         }
 
+        assert($token->last_used_at instanceof Time || $token->last_used_at === null);
```

Comment:
I don't know why but without this line, phpstan reports errors.

Reply:
How about adding `@property Time|null $last_used` to the entity?

Reply:
Although I don't like doing anything without understanding the reason, #417 seems to be waiting for this PR.